README.md

```diff
@@ -36,6 +38,33 @@ For reference, here's the time and memory usage that are required to transcribe
 
 *Executed with 8 threads on a Intel(R) Xeon(R) Gold 6226R.*
 
+
+### Distil-whisper
+
+| Implementation | Precision | Beam size | Time | Gigaspeech WER |
+| --- | --- | --- | --- | --- |
+| distil-whisper/distil-large-v2 | fp16 | 4 |- | 10.36 |
+| [faster-distil-large-v2](https://huggingface.co/metame/faster-distil-whisper-large-v2) | fp16 | 5 | - | 10.28 |
```

Comment:
Can you please update these link according to info in `utils.py`

Reply:
> Can you please update these links according to info in `utils.py`

thx, I updated the readme.

Reply:
Thanks for your contribution, I merge it now